man/systemd-logind.service.xml

```diff
@@ -82,6 +90,13 @@
       <listitem><para>Automatic spawning of text logins (gettys) on
       virtual console activation and user runtime directory
       management</para></listitem>
+
+      <listitem><para>Generate session ids. If auditing is available
```

Comment:
s/Generate session ids/Generating the session ID/

Also, it's not possible to say "before this module", because this is in the systemd-logind man page. Maybe say "before the `<citeref>pam_systemd</>` module"?